tests/e2e/network/service_annotations.go

```diff
@@ -475,8 +475,6 @@ var _ = Describe("Service with annotation", Label(utils.TestSuiteLabelServiceAnn
 			By("Cleaning up service and public IP")
 			err := utils.DeleteService(cs, ns.Name, serviceName)
 			Expect(err).NotTo(HaveOccurred())
-			err = utils.DeletePIPWithRetry(tc, publicIP, tc.GetResourceGroup())
```

Comment:
Why don't we need this?

Reply:
I think we need this when explicityly called `WaitCreatePIP()`?